devdox/tests/unit_test/test_doubles/app/repository/api_key_repository_test_doubles.py

```
@@ -0,0 +1,49 @@
+
+        matches = [hk for hk in hash_key_list if hk in self.existing_hashes]
+        return matches if matches else None
+
+    async def save_api_key(self, create_model: APIKeyCreate) -> Any:
+        if "save_api_key" in self.exceptions:
+            raise self.exceptions["save_api_key"]
+
+        self.received_calls.append(("save_api_key", create_model))
+
+        fake_entry = SimpleNamespace(
+            id= uuid4(),
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)